modules/openapi-generator/src/main/resources/dart2/api.mustache

```diff
@@ -192,7 +192,7 @@ class {{{classname}}} {
       {{/isArray}}
       {{^isArray}}
         {{#isMap}}
-      return {{{returnType}}}.from(apiClient.deserialize(_decodeBodyBytes(response), '{{{returnType}}}'));
+      return {{{returnType}}}.from(apiClient.deserialize(_decodeBodyBytes(response), '{{{returnType}}}') as Map<dynamic, dynamic>);
```

Comment:
we already know it's a map of things, what does knowing it is `<dynamic, dynamic>` bring?

Reply:
With the implicit casts false we get this:
![image](https://user-images.githubusercontent.com/7496871/102718391-a4c0f900-42c6-11eb-819d-c43d787fc512.png)

which means that we can't implicit cast a type to a more specialized type (`dynamic` to `Map<dynamic, dynamic>`)

Reply:
maybe we could just cast it instead of applying the map.from? :thinking: gonna try that in the https://dartpad.dev/

Reply:
just casting it would work:
```dart
void main() {
  print(foo({"a": 1}));
}

Map<String, int> foo(dynamic response) {
  return response as Map<String, int>;
}
```

Reply:
that's fine I'm cool with it, just was asking